docs/references.bib

```diff
@@ -3385,3 +3396,4 @@ @Article{         zorn1937
   doi           = {10.1090/S0002-9904-1937-06565-7},
   url           = {https://doi.org/10.1090/S0002-9904-1937-06565-7}
 }
+}
```

Comment:
I think this is a mistake?

Reply:
Yes, the bracket is redundant

Reply:
Hi, Thomas. All the problem is fixed. Please help me to proceed the merge! THX ;)